llumnix/metrics/base_metrics.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2024, Alibaba Group;
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from abc import ABC, abstractmethod
+
+from llumnix.metrics.variable import _REGISTRY, Status
+from llumnix.metrics.dumper import Dumper, DummyDumper
+from llumnix.instance_info import InstanceInfo
+
+class LlumnixMetrics(ABC):
+    def __init__(self):
+        self.instance_id = Status("instance_id")
```

Comment:
self.instance_id = Status("instance_id")
looks pretty confusing

Reply:
I followed brpc's naming conventions, where "status" is used to record and display a value，https://github.com/apache/brpc/blob/master/docs/cn/bvar_c%2B%2B.md#bvar-introduction